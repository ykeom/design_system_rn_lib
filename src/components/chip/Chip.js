import React, { Component } from 'react';
import { View, StyleSheet, Text } from 'react-native';

const styles = StyleSheet.create({
  root: {},
});

class Chip extends Component {
  render() {
    return (
      <View style={styles.root}>
        <Text>chip</Text>
      </View>
    );
  }
}

export default Chip;
