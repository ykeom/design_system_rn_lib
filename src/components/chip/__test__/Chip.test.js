import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Chip from '../Chip';

describe('Chip', () => {
  it('should render', () => {
    const component = shallow(<Chip />);

    expect(component.length).toBe(1);
    expect(toJson(component)).toMatchSnapshot();
  });
});
