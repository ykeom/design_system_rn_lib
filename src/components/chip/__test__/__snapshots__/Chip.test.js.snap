// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Chip should render 1`] = `
<View
  style={Object {}}
>
  <Text>
    chip
  </Text>
</View>
`;
